README.md

```diff
@@ -37,6 +37,209 @@
+
+Here’s how you do this.
+
+### 1. Declaring the protocol. 
+The protocol declaration will look like this:
+
+		protocol TextDescription {
+
+    			var textDescription: String { get }
+
+		}
+
+This declares a protocol and gives it a name — `TextClass Description`. The protocol declares that you will implement any and all properties and methods that it declares. In this case, there is only one property: `textDescription`. There can be several, and in some cases, you can make some of them optional but that’s for later on.
+
+> Protocol names are always capitalized just as Class names are. In the code shown here, remember that the protocol named TextDescription (capitalized) contains a variable textDescription that is lower-case. The protocol can contain many variables and even functions (functions in a protocol are for later on). The name of the protocol itself is separate from the names of the variables and functions within it.
+
+### 2.  Adopting the protocol.
+
+In your Class declaration, you *adopt* the protocol. Right now, the class declaration looks like this
+
+		class Person {
+
+Add the protocol so that it reads:
+
+		class Person: TextDescription
+		
+
+### 3. Conforming to the protocol 
+Conforming to a protocol means writing the code that implements it. In this case, you already have the code:
+
+In the Person class:
+
+		var textDescription: String {
+        		return "Name: \(self.name) \nHeight: \(self.height) \nFavorite Food: \(self.favoriteFood)"
+    		}
+
+For the Building class:
+		var textDescription: String {
+        		return "Address: \(self.address) \nHeight: \(self.height)"
+    		}
+    		
+This means that when you put the class declaration the protocol adoption, and the protocol conformance (implementation) together, here is what you get for Person:
+
+	class Person: TextDescription {
+
+    		// here are the properties
+    		var name: String
+    		var height: Double
+    		var favoriteFood: String
+
+		// this computed property implements the TextDescription protocol
+		var textDescription: String {
+			return "Name: \(self.name) \nHeight: \(self.height) \nFavorite Food: \(self.favoriteFood)"
+		}
+
+    		// here is the initializer method
+    		init(name: String, height: Double, favoriteFood: String) {
+        		self.name = name
+        		self.height = height
+        		self.favoriteFood = favoriteFood
+    		}
+	}
+
+Here is what you get for Building when the TextDescription protocol is adopted and implemented:
+		class Building: TextDescription {
+
+    			// here are the properties
+    			var address: String
+    			var height: Double
+			var textDescription: String {
+        			return "Address: \(self.address) \nHeight: \(self.height)"
+    			}
+
+    			// here is the initializer method
+    			init(address: String, height: Double) {
+        			address = name
+        			self.height = height
+   			}
+		}
```

Comment:
This code snippet seemed to get a little messed up in translation. As well, in the init we need to change `address = name` to `self.address = address`.

The snippet should look like this:

```swift
class Building: TextDescription {
    
    // here are the properties
    var address: String
    var height: Double
    var textDescription: String {
        return "Address: \(self.address) \nHeight: \(self.height)"
    }
    
    // here is the initializer method
    init(address: String, height: Double) {
        self.address = address
        self.height = height
    }
}
```